JS/Lesson22/tugba/style.css

```diff
@@ -0,0 +1,39 @@
+body {
+    font-family: Arial, sans-serif;
+}
+
+.accordion-container {
+    width: 60%;
+    max-width: 600px;
+}
+
+.accordion {
+    display: flex;
+    justify-content: center;
+    background-color: #f9f9f9;
+    color: #444;
+    cursor: pointer;
+    padding: 18px;
+    width: 80%;
+    text-align: left;
+    border-color:rgb(229, 229, 233);
+    outline: none;
+    transition: 0.4s;
+    border-radius: 10px;
+}
+
+.accordion:hover {
+    background-color: #619ecd;
+}
+
+.active, .accordion.active:hover {
+    background-color: #d21515;
```

Comment:
Red color feels a bit aggressive but again, it's a personal preference :) Feel free to use whatever you like!